catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -117,96 +125,29 @@ public void setHistorian(Historian historian) {
-            }
-            createRequest.getProperties()
-                    .put(PolicyPlugin.OPERATION_SECURITY, requestPolicyMap);
+        createRequest = queryOperations.setFlagsOnRequest(createRequest);
+        createRequest = validateCreateRequest(createRequest);
+        createRequest = validateLocalSource(createRequest);
 
-            for (AccessPlugin plugin : frameworkProperties.getAccessPlugins()) {
-                createRequest = plugin.processPreCreate(createRequest);
-            }
+        try {
+            createRequest = injectAttributes(createRequest);
+            createRequest = setDefaultValues(createRequest);
+            createRequest = updateCreateRequestPolicyMap(createRequest);
+            createRequest = processPrecreateAccessPlugins(createRequest);
 
             createRequest.getProperties()
                     .put(Constants.OPERATION_TRANSACTION_KEY,
                             new OperationTransactionImpl(OperationTransaction.OperationType.CREATE,
                                     Collections.emptyList()));
 
-            for (PreIngestPlugin plugin : frameworkProperties.getPreIngest()) {
-                try {
-                    createRequest = plugin.process(createRequest);
-                } catch (PluginExecutionException e) {
-                    LOGGER.info(
-                            "Plugin processing failed. This is allowable. Skipping to next plugin.",
-                            e);
-                }
-            }
-            validateCreateRequest(createRequest);
-
-            // Call the create on the catalog
-            LOGGER.debug("Calling catalog.create() with {} entries.",
-                    createRequest.getMetacards()
-                            .size());
-            if (Requests.isLocal(createRequest)) {
-                createResponse = sourceOperations.getCatalog()
-                        .create(createRequest);
-                createResponse = historian.version(createResponse);
-            }
-
-            if (catalogStoreRequest) {
-                CreateResponse remoteCreateResponse = doRemoteCreate(createRequest);
-                if (createResponse == null) {
-                    createResponse = remoteCreateResponse;
-                } else {
-                    createResponse.getProperties()
-                            .putAll(remoteCreateResponse.getProperties());
-                    createResponse.getProcessingErrors()
-                            .addAll(remoteCreateResponse.getProcessingErrors());
-                }
-            }
+            createRequest = processPreIngestPlugins(createRequest);
+            createRequest = validateCreateRequest(createRequest);
+            createResponse = getCreateResponse(createRequest);
+            createResponse = performRemoteCreate(createRequest, createResponse);
 
         } catch (IngestException iee) {
             INGEST_LOGGER.warn("Ingest error", iee);
```

Comment:
Maybe. I kept its level where it had been previously. It's possible that it was intentionally set at WARN in order to prevent overloading the ingest log. I'll see if I can figure that out.